1_basics.ipynb

```diff
@@ -25,20 +43,57 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# What is HSR?\n",
+    "\n",
+    "HSR (HSR: Human Support Robot) is one of the \"Toyota Partner Robots\" that support human activities, and it will assist independent living at home for people with disabilities.\n",
```

Comment:
"The HSR (HSR: Human Support Robot) is one of the \"Toyota Partner Robots\" that support human activities as well as assist independent living at home for people with disabilities.\n",